packages/shared/components/molecules/NftActivityDetails.svelte

```diff
@@ -12,7 +12,7 @@
     $: isTimelocked = activity?.asyncData?.timelockDate > $time
 </script>
 
-<nft-transaction-details class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0">
+<nft-transaction-details class="flex flex-auto w-full flex-col items-center justify-center space-y-3">
```

Comment:
Did you check these updates in desktop? 
Also, did the desktop team agree on the design changes made here, eg the y spacing?

Reply:
I talked with Jean today, since he was the last one who changed the line. The h-full leads to weird behaviour on mobile when you use the tab navigation, so we removed it. Couldn't see any problem on desktop with removing h-full. But changing the y-spacing isn't necessary actually, so I changed it back.